repls.org

```diff
@@ -1038,7 +1037,7 @@
    based REPL (using the ~figwheel-sidecar~ library), and the piggieback
    middleware is loaded, then Figwheel will automatically do the right thing.
 
-   Figwheel uses WebSockets just like [[Weasel][Weasel]].
```

Comment:
Same as above.

Reply:
Thanks @plexus 

It all make sense, I look at it from Github page which will not be linking correctly.
Thanks for pointing this out.